ci(helm-diff): add single binary scenario

kir4h opened the pull request:
**What this PR does / why we need it**:

Adds the single binary scenario to Helm diff CI. It makes sense since there are templates only rendered for it.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

Discussed on https://github.com/grafana/loki/pull/16930


**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [x] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

kir4h commented:
@trevorwhitney should we add the scenario?

jkroepke commented:
Hi @kir4h 

thanks for you PR. Could you please revert als chart unrelated changes and do a rebase?

trevorwhitney commented:
yeah I'm good with it if @jkroepke is

kir4h commented:
> Hi @kir4h
> 
> thanks for you PR. Could you please revert als chart unrelated changes and do a rebase?

Dropped the commit related to go mod tidy, rebased and force pushed. Will fix the PR if there is some issue in the tests

jkroepke commented:
LGTM
